addons/common/persistence/jdbc/src/test/java/org/kie/persistence/jdbc/JdbcProcessInstancesIT.java

```diff
@@ -96,7 +103,7 @@ void testBasicFlow() {
     }
 
     @Test
-    void testException() {
+    void testException(DataSource dataSource) {
```

Comment:
This is not a `ParameterizedTest` so no parameter necessary
```suggestion
    void testException() {
```